src/services/vault/client.rs

```diff
@@ -45,17 +45,23 @@ pub struct VaultClient {
 }
 
 impl VaultClient {
-    const LOGIN: &str = "/v1/auth/okta/login";
-    const VERIFY_TOKEN: &str = "/v1/auth/token/lookup-self";
-    const FETCH_SECRETS: &str = "/v1/secret/data";
-    const UPDATE_SECRET: &str = "/v1/secret/data";
+    pub const LOGIN: &str = "/v1/auth/okta/login";
+    pub const VERIFY_TOKEN: &str = "/v1/auth/token/lookup-self";
+    pub const FETCH_SECRETS: &str = "/v1/secret/data";
+    pub const UPDATE_SECRET: &str = "/v1/secret/data";
 
     pub fn new() -> Self {
         let client = reqwest::Client::new();
-        Self {
-            client,
-            base_url: "https://bunker.mindvalley.dev:8200".to_string(),
-        }
+        #[cfg(all(feature = "prod"))]
+        let base_url = "https://bunker.mindvalley.com".to_string();
```

Comment:
for prod, the Vault `base_url`  also the same as the dev environment. So we can just use the same url here.

Reply:
This is useful for testing, so in non prod environment we can use `WUKONG_DEV_VAULT_API_URL` to inject the vault url, e.g. mock server url.